tests/regression/wordpress-rule-exclusions-plugin/9507144.yaml

```
@@ -0,0 +1,110 @@
+---
+meta:
+  author: "Joost de Keijzer"
+  description: "Wordpress Rule Exclusions Plugin"
+rule_id: 9507144
+tests:
+  - test_id: 1
+    desc: Editing template part of a website i.e header or footer
+    stages:
+      - input:
+          dest_addr: 127.0.0.1
+          headers:
+            Host: localhost
+            User-Agent: OWASP CRS test agent
+            Accept: application/json, */*;q=0.1
+            Content-Type: application/json
+            x-http-method-override: PUT
+          port: 80
+          method: POST
+          version: "HTTP/1.1"
+          # URI is actually sent with double slashes
+          uri: /index.php?rest_route=/wp/v2/template-parts/twentytwentyfour//header&_locale=user
+          # Data is sent with some special characters escaped
+          data: |
```

use the folded yaml scalar to preserve the structure of the payload, this will send the payload literally.
```suggestion
          data: >-
```